app/core/models/ocelot/stanzas/ChoiceStanza.scala

```diff
@@ -67,46 +70,61 @@ sealed trait ChoiceTest {
   val right: String
   def eval(labels: Labels): Boolean
   def value(arg: String, labels: Labels): String = labelReference(arg).fold(arg)(ref => labels.value(ref).getOrElse(""))
-  def op(f: (BigDecimal, BigDecimal) => Boolean, g: (String, String) => Boolean, labels: Labels): Boolean = {
+  def moreThanOrEquals(date: LocalDate, other: LocalDate): Boolean = date.compareTo(other) >= 0
```

Comment:
think these fns dont belong here, the ChoiceTest trait is common to all tests without any type or test specific fns, suggest using an anonymous fn as other tests i.e. _.compareTo(_) >= 0 and _.compareTo(_) <= 0

Reply:
Thanks